packages/react/src/component-tests/IcBreadcrumb/IcBreadcrumb.cy.tsx

```diff
@@ -22,6 +23,9 @@ import {
   NOT_EXIST,
 } from "../utils/constants";
 import { setThresholdBasedOnEnv } from "../../../cypress/utils/helpers";
+import { IcBreadcrumb, IcBreadcrumbGroup, IcLink } from "../../components";
```

Comment:
I think you can remove all these added imports, as they are not used. The ones that are needed are already in the testdata file